docker/packager/binary/Dockerfile

```diff
@@ -83,5 +83,8 @@ RUN export CODENAME="$(lsb_release --codename --short | tr 'A-Z' 'a-z')" \
         --yes --no-install-recommends \
     && apt-get clean
 
+# for external_symbolizer_path
+RUN ln -s /usr/bin/llvm-symbolizer-15 /usr/bin/llvm-symbolizer
```

Comment:
is it compatible with llvm-14?

Reply:
I'm sure that it is.

Reply:
Agree. I haven't checked but I'm pretty sure that it should work.